types/chai-http/index.d.ts

```diff
@@ -3,12 +3,15 @@
 // Definitions by: Wim Looman <https://github.com/Nemo157>
 //                 Liam Jones <https://github.com/G1itcher>
 //                 Federico Caselli <https://github.com/CaselIT>
+//                 Bas Luksenburg <https://github.com/bas-l>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.3
 
 /// <reference types="node" />
 /// <reference types="chai" />
 
+import request = require('superagent');
```

Comment:
Just a nitpick: can you use an es6 import? Like `import * as request from 'superagent';`

Reply:
Done :+1: